bin/history+

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
 function __history+() {
-    local text="${1:-$(date +%Y-%m-%d)}"
+    local SEARCHTEXT=$(date +%Y-%m-%d)  # Default search argument
+    local COLOROUT=0            # Define default to print out with no color
+    local HEADEROUT=0           # Define default to print out with no header
     local B0=$(tput setab 0)    # Set background color to black
```

Comment:
There are two options here:

1. Move these definitions to the block which needs it, I mean: if coloured output is not needed, we should not define variables which will never be used.

2. Move these definitons to bashrc if your intent is presenting coloured text in other commands too. Note that commands like `ls`, `grep`, `rg`, etc provide such functionality of coloured output too, and I'm curious to understand what would be the defaults for these commands. I mean: these commands may be taking the definitions from some sort of database or configuration file or whatever. Ideally, you should import color definitions from the same source, so that the same definitions are valid not only on those commands, but also history+.